.ci/env/apt.sh

```diff
@@ -27,14 +31,7 @@ function add_repo {
 }
 
 function install_dpcpp {
-    sudo apt-get install                    \
-        intel-oneapi-common-vars            \
-        intel-oneapi-common-licensing       \
-        intel-oneapi-tbb-devel              \
-        intel-oneapi-dpcpp-cpp-compiler     \
-        intel-oneapi-dev-utilities          \
-        intel-oneapi-libdpstd-devel         \
-        cmake
+    sudo apt-get install -y intel-dpcpp-cpp-compiler-2023.2.1
```

Comment:
Just a thought which crossed my mind: would we have to actively keep track of the dpcpp complier version in the future with this change? (i.e. change it in the future via PR to 2024.0 etc.) since previously the call to intel-oneapi-dpcpp-cpp-compiler probably masked versioning there (I assume?). It looks like manual versioning was done for the other software dependencies (bazelisk etc.).

Reply:
this particular line is same as we have currently  - https://github.com/oneapi-src/oneDAL/blob/master/.ci/scripts/install_dpcpp.sh#L24 

The problem is that with other places like bazel or workspace dependencies we will have PR from renovate. But with this manually defined deps we have to PR on our own